hasidiluk/drupal
# Form build

### What does this pull request fix/change?

This is a test form that submit customer details.

### Testing steps

- Checkout to this branch.
- Clear drupal cache.
- Navigate to http://drupalvm.test/user/1/spy_glass#
- Customer data submission form should appear.
- Make sure form submit data without having any errors.
